src/main/java/lottoModel/LottoMaker.java

```diff
@@ -0,0 +1,30 @@
+package lottoModel;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoMaker {
+
+    static List<List<Integer>> lottoSaved = new ArrayList<>();
```

Comment:
default 접근 제어자를 사용하신 이유가 있을까요? private 접근 제어자가 적절해보입니다.
```suggestion
    private static List<List<Integer>> lottoSaved = new ArrayList<>();
```
만약, 해당 리스트를 사용하고 싶다면, get메소드를 추가하는 것을 추천합니다